map_name: "maps/Nominal_map.json"
# map_name: "maps/Generalization_map.json"


video_source : 'http://team10.local:8890/video'
driving_websocket : "ws://team10.local:8887/wsDrive"
lidar_source : 'http://team10.local:8890/lidar'

waypoint_threshold : 2.5
angle_threshold : 0

size_factor : 7.33
x_map_shift : 48
y_map_shift : 50
angle_shift : 0

simulator_ip : "127.0.0.1"
simulator_port : 9091

default_throttle_multiplier : 0.365

model_path : "models/ft_model.h5"

fixed_throttle : True

initial_lane : "left"
obstacle_distance_threshold : 9

default_target_speed : 0.4

car_kp : 0.005
car_ki : 0
car_kd : 0.1

sim_kp : 0.5
sim_ki : 0.0001
sim_kd : 0.1

